ui/.eslintrc.js

```diff
@@ -74,6 +72,9 @@ module.exports = {
     '@typescript-eslint/prefer-optional-chain': 'warn',
     '@typescript-eslint/unified-signatures': 'warn',
     // '@typescript-eslint/member-ordering': ['warn', { 'classExpressions': ['method', 'field'] }],
+    // '@typescript-eslint/prefer-function-type': 'warn',
```

Comment:
I judged these to be more "functional" than "style" so I moved them to this section